kafka-client/src/main/java/dev/responsive/kafka/internal/db/CassandraKeyValueTable.java

```diff
@@ -244,14 +305,20 @@ public byte[] get(
 
   @Override
   public KeyValueIterator<Bytes, byte[]> range(
-      final int partition,
+      final int kafkaPartition,
       final Bytes from,
       final Bytes to,
       final long minValidTs
   ) {
+    // TODO: need to scan a range of sub-partitions to correctly implement range
```

Comment:
should we throw an exception in the meantime?

Reply:
Probably, yeah!

Note: we were already throwing an exception in the layer above, ie `PartitionedOperations`, so it's not like this could accidentally have been invoked. That said, I do think it makes sense to just implement the `PartitionedOperations#range` method and move the exception to the `CassandraKeyValueTable#range`, since that's where the fundamental limitation/missing functionality is. Same with all

(I did leave the UnsupportedOperationExceptions in PartitionedOperations for the reverseRange/reverseAll, however)